def test_this_func():
    print('This is running and expecting a raise Exception')
    # raise Exception('haha this fails')

var = 5
for i in range(var):
    print i


def not_call():
    he = 'this code will not be called'
    return 'whuat?'




def OUCHhah():
    a = 'bad naming here...jeeez'
    b = 'aaaaa'
    c= 121221
    de='adad' + 'adasddsds'
    return 124






test_this_func()


print 'whaaat?'



